tests/sort-demo/TestBoard.pas

```diff
@@ -102,36 +145,69 @@ procedure TestTBoard.TestSwapTwoLastValues;
 procedure TestTBoard.TestSwapNegativeIndexes;
 begin
   // TODO: [TeamC] Zweryfkować czy swap dwóch ujemnych indeksów rzuca wyjątkiem
+  FBoard.GenerateData(10);
+  Self.StartExpectingException(Exception);
```

Comment:
Exception to dość ogólny wyjątek

Reply:
👍 brawo Ty

Znowu uwaga komunikacyjna: słowo `dość` jest mało konkretne i może być odebrane negatywnie, ale nie o to chodziło. Ten sam komunikat bez słowa `dość` jest bezpieczniejszy. Ja bym napisał:
* `Exception to ogólna klasa`
Jakbym chciał być bardzo grzeczny to:
* `Ja bym łapał tutaj konkretną klasę wyjątku rzucaną przez TBoad, a nie ogólny Exception`